docs/modules/cp-subsystem/pages/cp-subsystem.adoc

```diff
@@ -118,6 +118,20 @@ For example, a cluster has three CP members and three CP groups, each CP member
 Raft leader for only one CP group. If one more CP group is created, then one of
 the CP members becomes the Raft leader for two CP groups.
 
+=== CP members priority
+However, there are cases/scenarios when some CP members shouldn't act as a leader. For example, a server with high load would not be a good leader, or in a WAN deployment, servers in a primary datacenter may be preferred in order to minimize the latency between clients and the leader.
+
+To ensure the availability of the CP subsystem, we cannot prevent some members from being elected as a leader. But we can ask to transfer leadership from the existing leader to other CP members.
+
+To do this, and in addition to an existing rebalancing strategy, you can set up priority-based leadership.
+
+Each CP member can be assigned a priority - a positive or negative integer number. Initially, any CP member with any priority can take the leader's role. But the periodically running leadership rebalancing task will transfer the leadership from CP members with the lowest priorities within each CP group.
+So, eventually, all CP group leaders will be transferred to CP members with higher priorities.
+
+To configure the CP member priority, use the xref:configuration.adoc#cp-member-priority[`cp-member-priority`] option.
+
+NOTE: Current limitations: We do not address an issue when a CP group can only be formed from CP members with the lowest priority and then promote CP members with higher priorities.
```

Comment:
WARNING: CP member prioritization does not work when a CP group initially can only be formed from CP members with the lowest priorities, and then promote CP members with higher priorities. For example, if you start the cluster with three CP Members each having priority `0` and later add or promote new members with priorities `100`, you will still have a CP group with members having priorities `0`.